dev-docs/lit-release-process.md

```diff
@@ -0,0 +1,148 @@
+
+2. Click on the link "Review pending deployments" which will navigate you to a GitHub Action page for the Release workflow. Then click "Review deployments" and approve the pending deployment.
+
+![GitHub actions UI for the Release showing a "Review deployments" button](./images/lit-release-process/release-github-action-review-pending.png)
+
+![GitHub actions UI modal for approving a pending release](./images/lit-release-process/review-pending-release-modal-ui.png)
+
+3. The GitHub release action will now automatically generate a "Version Packages" PR.
+
+<blockquote>
+If a Version Packages pull request has not been created, an error may have occurred in the GitHub action run. Look at the full logs for root cause.
+
+Possible failures previously encountered: A malformed changest file referencing a package name that doesn't exist.
+The fix was to commit the fixed changeset file to main, and re-run the release process on that new commit with the fixed changeset.
+
+</blockquote>
+
+![A screenshot of a GitHub pull request title "Version Packages"](./images/lit-release-process/version-packages-pr.png)
+
+4. Request code reviews from members of the team familiar with the packages that will be released.
+
+   1. The purpose of this pull request is to:
+      1. Delete all the changeset files, moving their contents into the various Changelog files.
+      1. Bump the `window.litElementVersions` global.
+      1. Bump version numbers in package.json files.
+      1. Bump dependency ranges in related package.json files if needed. (Only expected in minor or major releases).
+   1. Errors to look for:
+      1. When a new package is created and released, it is easy to accidentally have the version in the package.json set to vesion 1.0.0, resulting in Changeset bumping the version to 2.0.0. Fix is to create the package at version 0.0.0.
+      1. Major version releases or minor features may require further scrutiny as major releases mark major events and require coordination.
+      1. Changelog updates to `lit` core libraries such as `lit-html` should be relected in the umbrella `lit` package.
+      1. Internal private or not ready to be released packages should not get changelogs or version bumps.
+   1. Fix errors by closing the automated "Version Packages" pull request. Commit changes to `main` branch and re-run the release process.
+
+5. Once "Version packages" is approved, (provided no new changesets were merged to `main` in the meantime), squash and merge.
+
+<blockquote>
+
+⚠️ If a commit to `main` occurred while the "Version Packages" PR was open and a new changeset was created then do not squash and merge.
+Instead re-run version packages to update the PR with the most recent changes. Otherwise the following steps will not publish a release and will instead create another "Version Packages" PR integrating the new changeset file.
+
+A publish only occurs when there are no unreleased changeset files on `main`.
+
+</blockquote>
```

Comment:
Do you anticipate these docs would mostly be consumed from within Github? What do you think about trying this fancy github blockquote feature? https://github.com/orgs/community/discussions/16925

Reply:
Ooooh very great suggestion - will apply this!

Reply:
Done